.github/workflows/unit-test.yml

```diff
@@ -21,8 +21,26 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
 
+      - name: Checkout python env
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Checkout java env
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+
+      - name: Setup-hdfs env
+        uses: beyondstorage/setup-hdfs@master
```

Comment:
All tests that need to connect with real hdfs cluster should be removed into integration tests instead.